scenarios/14d_Avenging_Ruen.cfg

```diff
@@ -9,11 +9,11 @@
     {EXTRA_SCENARIO_MUSIC theme_of_a_new_order.ogg}
     {EXTRA_SCENARIO_MUSIC weight_of_revenge.ogg}
 
-    {TURNS 28 24 21}
+    {TURNS 36 29 22}
```

Comment:
yeah the first time I played through this the turn limit was still 28 and I had to increase it by debug mode to finish; first I increased it to 32 and that still wasn't enough, so then I went with 36 just because that was the next nice round number, but I did finish on turn 33 anyways, so 33 could work. (On my next two playthoughs I finished on turns 20 and 18, for reference)

Reply:
I went with `{35 31 27}` in 4bdbac0; hope that's better